index.js

```diff
@@ -50,7 +49,7 @@ module.exports = {
     if (options.coreOnly) {
       appboyIncludes = ['appboy.core.min.js'];
     } else {
-      appboyIncludes = ['appboy.min.js', 'appboy.min.css'];
```

Comment:
According to the [CHANGELOG](https://github.com/Appboy/appboy-web-sdk/blob/master/CHANGELOG.md#200), these styles are automatically loaded in 2.x